homeassistant/components/openuv/__init__.py

```diff
@@ -210,7 +210,7 @@ def __init__(self, client, binary_sensor_conditions, sensor_conditions):
             if data.get('from_time') and data.get('to_time'):
                 self.data[DATA_PROTECTION_WINDOW] = data
             else:
-                _LOGGER.error(
+                _LOGGER.info(
```

Comment:
I think that we should set it to `debug` as it's the outcome would be visible in the frontend if used.